src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,8 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (!a || !b || !c || !d) && (((a && b) && (!c && !d))
+                || ((!a && !b) && (c && d))
+                || ((a != b) && (c != d)) ? true : false);
```

Comment:
```suggestion
                || ((a != b) && (c != d)));
```
ternary operator is redundant here, try to remove it and check if solution works